doc/whats_new.rst

```diff
@@ -156,6 +156,8 @@ API
 
     - The C wrapper :func:`mne.do_forward_solution` has been deprecated in favor of the native Python version :func:`mne.make_forward_solution` by `Eric Larson`_
 
+    - The ``event`` parameter of :func:`mne.epochs.EpochsArray` is set by default to chronological time-samples and event values of 1, by `Jean-Remi King`_
```

Comment:
I think "of 1" actually works slightly better here -- removing the intermediate bits it's either the current "event parameter ... is set by default to ... event values of 1" or "event parameter .. is set by default to ... event values to 1". The former reads better to me.